outport/notifier/httpClient.go

```diff
@@ -84,5 +84,9 @@ func (h *httpClient) Post(
 		return err
 	}
 
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("HTTP status code: %d, %s", resp.StatusCode, http.StatusText(resp.StatusCode))
+	}
```

Comment:
please also log (WARN or ERROR) `string(resBody)` in case of a non-200 response. This will allow us to see what went wrong easier in the future

Reply:
added warn log meesage